client/blocks/credit-card-form/index.jsx

```diff
@@ -305,40 +302,29 @@ const CreditCardForm = createReactClass( {
 					</div>
 					{ this.renderUsedForExistingPurchases() }
 				</Card>
-
 				<CompactCard className="credit-card-form__footer">
-					<em>{ this.props.translate( 'All fields required' ) }</em>
-
+					<em>{ translate( 'All fields required' ) }</em>
 					<FormButton disabled={ this.state.formSubmitting } type="submit">
-						{ this.state.formSubmitting
-							? this.props.translate( 'Saving Card…', {
-									context: 'Button label',
-									comment: 'Credit card',
-								} )
-							: this.props.translate( 'Save Card', {
-									context: 'Button label',
-									comment: 'Credit card',
-								} ) }
+						{ this.getSubmitButtonText() }
```

Comment:
why did we extract this out? usually we _do_ tend to prefer inlining inside of `render()` but that's not a hard rule

Reply:
Prettify kept shifting the spacing around post-commit. I thought I could wrangle an acceptable return statement elsewhere (which I couldn't), then found the abstraction more pleasing to the eye. 

> usually we do tend to prefer inlining inside of render()
Good to know. No hassle to whack it back in.